index.md

```diff
@@ -72,3 +72,7 @@ Once operational, Helen finds a verification agent, Lynn, who was able to inspec
 ## UC15 Verified Impact
 
 Lynn is a verification agent for REC2022. She is called by HydroElec to audit their new production facility. After reviewing the certified proposal, she visits the HydroElec site to inspect and verify both the capacity of the system under operating conditions and the security of the automated smart meters. She uses a standard set of criteria established by the UN to record her observations and make a determination about the project’s compliance with both its certified proposal and UN requirements. After finding everything in order, Lynn issues a verification credential with a six month expiry and a maximum capacity of 20 billion kWh, which she gives to HydroElec. The credential references both the original proposal and its certification using machine-verifiable identifiers to ensure the authenticity of the verification.
+
+## UC22 DAO Discipline
+
+The Ndugu Vijana Chama’s CIC currency has reached 80 cents on the shilling for the third time this year. The other Chamas need to decide on whether to take disciplinary action, rehabilitative action, or to expel the group from the Sarafu Network. They have a deliberation. There are 2 proposals for rehabilitation, 1 proposal for disciplinary action, and 1 proposal to eject them. Each chama has a weighted vote based on the number of people in the chama, and based on the status of the chama (reputation) in the network. Each Chama receives 10 points to distribute among the 5 proposals on the table. They can give all votes to one or divide their points between the different proposals depending on the strength of each proposal.
```

Comment:
Is it 80 cents on the dollar haha?

And some context about why this is bad might be useful.

Also **C**hama throughout

Reply:
@jandrieu Hey there! Do you want to update based o Will's comments?